README.md

```diff
@@ -31,6 +31,8 @@ Build
 
 ### Requirements
 
+On some systems: `findutils` `gawk` `zip` `wget` need to be manually installed. Busybox won't work.
```

Comment:
"some systems" with busybox sounds a lot like Alpine Linux. If you've tested it there, just write Alpine Linux